hedera-node/src/test/java/com/hedera/services/files/TieredHederaFsTest.java

```diff
@@ -194,9 +184,9 @@ void relevantInterceptorsGetPostCb() {
 
 		given(highInterceptor.preUpdate(fid, newContents))
 				.willReturn(
-						new AbstractMap.SimpleEntry<>(ResponseCodeEnum.OK, true));
+						new AbstractMap.SimpleEntry<>(ResponseCodeEnum.SUCCESS, true));
 		given(lowInterceptor.preUpdate(fid, newContents)).willReturn(
-				new AbstractMap.SimpleEntry<>(ResponseCodeEnum.OK, true));
+				new AbstractMap.SimpleEntry<>(ResponseCodeEnum.SUCCESS, true));
```

Comment:
Are we changing the outcome status from `OK` to `SUCCESS`? (successful operations now return success and previously they were returning `ok`?

Reply:
As far as I understood what was happening, the file system did whatever operation was called, then returned OK status, which was then checked and returned SUCCESS if the status from the FS was OK.